### PR TITLE
Feature: Event image - storage and upload

### PR DESCRIPTION
**Pull request by ristovska1606**

### CODE REVIEW

**Comment by SazdovaEkaterina** [comment]
[File Storage - Materials & Events](https://trello.com/c/e72CSN5N/33-file-storage-materials-events)

**Comment by ristovska1606** [comment]
[File / Image Upload - Events](https://trello.com/c/sGxesAQ7/58-file-image-upload-events)

File: src/backend/TeachersSideAPI/TeachersSideAPI/Program.cs [hunk 1]
```diff
@@ -97,6 +112,8 @@
 
 app.UseAuthorization();
 
+app.UseStaticFiles();
```
Comment (ristovska1606) [inline src/backend/TeachersSideAPI/TeachersSideAPI/Program.cs:115]: remove this

File: src/backend/TeachersSideAPI/TeachersSideAPI/Service/Implementation/EventService.cs [hunk 2]
```diff
@@ -14,12 +14,14 @@ public class EventService : IEventService
     private readonly IEventRepository _eventRepository;
     private readonly IMapper _mapper;
     private readonly UserManager<Teacher> _userManager;
+    private readonly IWebHostEnvironment _env;
 
-    public EventService(IEventRepository eventRepository, IMapper mapper, UserManager<Teacher> userManager)
+    public EventService(IEventRepository eventRepository, IMapper mapper, UserManager<Teacher> userManager, IWebHostEnvironment env)
```
Comment (ristovska1606) [inline src/backend/TeachersSideAPI/TeachersSideAPI/Service/Implementation/EventService.cs:19]: check this one

**Review by SazdovaEkaterina — APPROVED** [review]

**Pull request merged**